tests/integration/schema_registry/test_json_serializers.py

```diff
@@ -150,7 +149,7 @@ def test_json_record_serialization_incompatible(kafka_cluster, load_file):
               "contractRate": 1250,
               "trades": ["mason"]}
 
-    with pytest.raises(SerializationError,
+    with pytest.raises(KafkaException,
```

Comment:
Serializers should raise Key|ValueSerializerError, inheritance is:
Key|ValueSerializerError -> SerializerError -> KafkaException (has KafkaError as args 0) -> Exception